service-bus.api.md

```diff
@@ -0,0 +1,256 @@
+## API Report File for "@azure/service-bus"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AmqpMessage } from '@azure/core-amqp';
+import { DataTransformer } from '@azure/core-amqp';
+import { delay } from '@azure/core-amqp';
+import { Delivery } from 'rhea-promise';
+import Long from 'long';
+import { MessagingError } from '@azure/core-amqp';
+import { OperationOptions } from '@azure/core-auth';
+import { RetryOptions } from '@azure/core-amqp';
+import { TokenCredential } from '@azure/core-amqp';
+import { TokenType } from '@azure/core-amqp';
+import { WebSocketImpl } from 'rhea-promise';
+import { WebSocketOptions } from '@azure/core-amqp';
+
+// @public
+export interface CorrelationFilter {
+    contentType?: string;
+    correlationId?: string;
+    label?: string;
+    messageId?: string;
+    replyTo?: string;
+    replyToSessionId?: string;
+    sessionId?: string;
+    to?: string;
+    userProperties?: any;
+}
+
+// @public
+export interface CreateBatchOptions extends OperationOptions {
+    maxSizeInBytes?: number;
+    retryOptions?: RetryOptions;
```

Comment:
I do recall that this was added upon my insistence in another PR, but I believe the retry options shouldn't be here.
We do not need to have `retryOptions` customizable by the user in every method. At the bare minimum it should appear in the client constructor. As an extra, it may appear in the getSender/getReceiver methods.